indico/modules/events/management/controllers.py

```diff
@@ -134,3 +140,49 @@ def _checkParams(self, params):
     def _process(self):
         objects = self.obj.get_non_inheriting_objects()
         return jsonify_template('events/management/non_inheriting_objects.html', objects=objects)
+
+
+class RHEventProtection(RHConferenceModifBase):
+    """Show event protection"""
+
+    def _process(self):
+        form = EventProtectionForm(obj=FormDefaults(**self._get_defaults()), event=self.event_new)
+        if form.validate_on_submit():
+            update_event(self.event_new, {'protection_mode': form.protection_mode.data})
+            # TODO replace with self.event_new.is_self_protected when ACLs are in new DB
+            from indico.core.db.sqlalchemy.protection import ProtectionMode
```

Comment:
Importing stuff from `indico.core` and `indico.util` should be safe; if this causes an import cycle it should probably be fixed "on the other side"